### PR TITLE
[Group] Add Group Chat tab to existing Group page

### PR DESCRIPTION
**Pull request by rkdavidson**
# Related Issue
- [**🏕 Restructure Groups to more closely match mockups for chat**](https://3.basecamp.com/3926363/buckets/17566548/todos/3111116484)

# Changes or Updates
- Adds temporary Chat tab to Group page so it can be shipped independently of wider Group page layout overhaul

# Tradeoffs/Caveats
This isn't a long term layout/solution, just a temporary one so we _can_ ship Group Chat independently of larger Group page layout overhaul work. The tabs are slightly clunky/unrefined when closely examined, but, this is a brief interim state on the way to a much better one shortly.

**🛑 Note** we cannot merge this until the `group-chat` api branch is merged.

#Test Instructions
1. Ensure sandbox mode is **disabled** in your `.env` (`REACT_APP_ENABLE_SANDBOX=`)
1. Ensure you are on the `group-chat` branch on local **🛰️ api**
1. Go to a group page
1. 🔸 Verify you see an **About** and **Chat** tab, and that they work as you'd expect
1. To simulate chat being disabled... open src/groups/GroupContentItemConnected/index.js and remove line 101, where the `chatChannelId` prop is passed down into `<GroupContentItem>`
1. 🔸 Verify the **Chat** tab is gone, and that only **About** is displayed and it is displayed without the blue bottom border

# Screenshots

|State|Screenshot|
|---|---|
|Chat enabled - About Tab||
|Chat enabled - Chat Tab||
|Chat disabled||
|Interacting with the Tabs||

# Tests
- [ ] needed snapshots updated
- [ ] mobile tested
- [ ] latest update from master

### CODE REVIEW

File: src/groups/GroupContentItemConnected/GroupContentItem.js [hunk 1]
```diff
@@ -40,7 +75,11 @@ const GroupContentItem = ({
   title,
   userName,
   videoCall,
+  chatChannelId,
 }) => {
+  const [activeTab, setActiveTab] = useState('about');
+
+  console.log('[rkd] chatChannelId:', chatChannelId);
```
Comment (lmommaerts) [inline src/groups/GroupContentItemConnected/GroupContentItem.js:82]:
```suggestion
```

File: src/groups/GroupContentItemConnected/GroupContentItem.js [hunk 2]
```diff
@@ -40,7 +75,11 @@ const GroupContentItem = ({
   title,
   userName,
   videoCall,
+  chatChannelId,
```
Comment (lmommaerts) [inline src/groups/GroupContentItemConnected/GroupContentItem.js:78]: @rkdavidson This needs added in propTypes

**Review by lmommaerts — CHANGES_REQUESTED** [review]
@rkdavidson We'll need to pass `chatChannelId` to `GroupContentItem`:
https://github.com/christfellowshipchurch/web-app/blob/master/src/groups/GroupContentItemConnected/index.js#L100
```
      ...
      videoCall={get(content, 'videoCall')}
      chatChannelId={get(content, 'chatChannelId')}
    />
```

File: src/groups/GroupContentItemConnected/GroupContentItem.js (hunk 2 above)
Comment (rkdavidson) [reply src/groups/GroupContentItemConnected/GroupContentItem.js:78]: Fixed in 66adedb

**Review by lmommaerts — APPROVED** [review]
Looks good! 👍

**Pull request merged**